### PR TITLE
cmake: using LayerUtils repository

### PR DESCRIPTION
**Pull request by christophe-lunarg**

### CODE REVIEW

File: scripts/known_good.json [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+            "commit": "main",
+            "deps": [
+                {
+                    "var_name": "VULKAN_HEADERS_INSTALL_DIR",
+                    "repo_name": "Vulkan-Headers"
+                },
+                {
+                    "var_name": "GOOGLETEST_INSTALL_DIR",
+                    "repo_name": "googletest"
+                }
```
Comment (juan-lunarg) [inline scripts/known_good.json:26]: Pass `-D UTILITY_LIBRARIES_BUILD_TESTS=OFF` instead.

This avoids having to deal with this dependency.

File: CMakeLists.txt [hunk 2]
```diff
@@ -34,6 +34,8 @@ get_filename_component(VVL_INCLUDE_DIR ${VVL_INCLUDE_DIR} DIRECTORY)
 # This reflects the current structure of the VulkanSDK.
 target_include_directories(Vulkan::Headers INTERFACE ${VVL_INCLUDE_DIR})
 
+find_package(VulkanLayerUtils REQUIRED CONFIG)
```
Comment (juan-lunarg) [inline CMakeLists.txt:37]: `VulkanLayerUtils` doesn't have `find_package` support yet.

**Review by juan-lunarg — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (christophe-lunarg) [reply CMakeLists.txt:37]: Ah, I wasn't undertanding why it wasn't working! \o/

Do you have a reference for how it's done in another repository?

File: scripts/known_good.json (hunk 1 above)
Comment (christophe-lunarg) [reply scripts/known_good.json]: Yes that's great, thanks!

**Comment by juan-lunarg** [comment]
Closing in favor of https://github.com/KhronosGroup/Vulkan-Profiles/pull/399

**Pull request closed without merging**